charts/rancher-istio-1.8/rancher-istio/1.8.601/charts/kiali/templates/dashboards/envoy.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if (include "kiali-server.isDashboardEnabled" .) }}
```

Comment:
All these istio 1.8.601 charts need to be deleted, there were added by mistake and not removed. 
https://github.com/rancher/charts/tree/dev-v2.5/assets/rancher-istio-1.8
https://github.com/rancher/charts/tree/release-v2.5/assets/rancher-istio-1.8


Can you scan the charts folder versions being added here and make sure they line up with the assets and the "new charts" issue. 
cc @jiaqiluo / @paynejacob